### PR TITLE
feat: manage text with react-intl

### PR DESCRIPTION
**Pull request by MatsThieme**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -42,6 +42,6 @@ services:
   smtp:
     image: ixdotai/smtp:latest
     ports:
-      - 2525:25
+      - "2525:25"
```
Comment (Headscracher) [inline docker-compose.yaml]: Is there a specific reason you changed that?

**Review by Headscracher — APPROVED** [review]
Looking really good so far. Great work 👍

File: docker-compose.yaml (hunk 1 above)
Comment (MatsThieme) [reply docker-compose.yaml]: Docker couldn't run in wsl 2 Ubuntu on windows without the quotes. Shouldn't be part of this MR

Comment (Headscracher) [reply docker-compose.yaml]: okay just remove it in this PR and I'll add in general in a seperate commit

**Pull request merged**